Source/Flow/Private/Nodes/Route/FlowNode_NamedRerouteDeclaration.cpp

```diff
@@ -0,0 +1,145 @@
+﻿// Copyright https://github.com/MothCocoon/FlowGraph/graphs/contributors
+
+#include "Nodes/Route/FlowNode_NamedRerouteDeclaration.h"
+
+#include "FlowAsset.h"
+#include "Nodes/Route/FlowNode_NamedRerouteUsage.h"
+
+UFlowNode_NamedRerouteDeclaration::UFlowNode_NamedRerouteDeclaration(const FObjectInitializer& ObjectInitializer)
+	: Super(ObjectInitializer)
+{
+#if WITH_EDITOR
+	Category = TEXT("Named Reroute");
+	NodeStyle = EFlowNodeStyle::Custom;
+#endif
+	NodeTitle = TEXT("Named Reroute");
+	AllowedSignalModes = {EFlowSignalMode::Enabled, EFlowSignalMode::Disabled, EFlowSignalMode::PassThrough};
+}
+
+void UFlowNode_NamedRerouteDeclaration::ExecuteInput(const FName& PinName)
+{
+	TriggerFirstOutput(true);
+	for (const UFlowAsset* FlowAsset = GetFlowAsset();
+	const auto Usage : FlowAsset->FindNamedRerouteUsages(DeclarationGuid))
+	{
+		if (Usage)
+		{
+			Usage->TriggerFirstOutput(true);
+		}
+	}
+}
+
+void UFlowNode_NamedRerouteDeclaration::PostInitProperties()
```

Comment:
PostInitProperties(), PostLoad(), PostDuplicate(), GetNodeTitle() should be wrapped with #if WITH_EDITOR